com_028_rw1_effect_of_ag_prices_on_commodity_prices/README.md

```diff
@@ -0,0 +1,12 @@
+## Effect of Agricultural Policies on Commodity Prices Dataset Pre-processing
+This file describes the data pre-processing that was done to [the Nominal Rate of Protection](http://www.ag-incentives.org/indicator/nominal-rate-protection) for [display on Resource Watch](https://resourcewatch.org/data/explore/641c0a35-f2e5-4198-8ed9-576ea7e9685a).
+
+The data source provided the dataset as one csv file and one README file. The csv file was reformated to upload to Carto. We converted the years in the 'year' column to datetime objects and store them in a new column 'datetime'. We removed the column 'notes' since it contains an index without descriptions. The column names were converted to lowercase before we uploaded the data to Carto. 
```

Comment:
```suggestion
The data source provided the dataset as one csv file. Years in the 'year' column were converted to datetime objects and stored in a new column 'datetime'. The 'notes' column was removed since it only contains indexes instead of actual data. The column names were converted to lowercase before we uploaded the data to Carto. 
```